src/roaring64.c

```diff
@@ -803,6 +803,50 @@ bool roaring64_bitmap_run_optimize(roaring64_bitmap_t *r) {
     return has_run_container;
 }
 
+/**
+ *  (For advanced users.)
+ * Collect statistics about the bitmap
+ */
+void roaring64_bitmap_statistics(const roaring64_bitmap_t *r,
+                                 roaring64_statistics_t *stat) {
+    memset(stat, 0, sizeof(*stat));
+    stat->cardinality = roaring64_bitmap_get_cardinality(r);
```

Comment:
nit: since we get the cardinality of each container, we could move this into the loop over containers, rather than going through the containers twice.

Reply:
done